Description: Change path to expect in secure-card example script

This fixes the lintian warning:
I: ppp: example-wrong-path-for-interpreter
  usr/share/doc/ppp/examples/scripts/secure-card
  (#!/usr/local/bin/expect != /usr/bin/expect)

Author: Chris Boot <bootc@debian.org>
Forwarded: not-needed
Last-Update: 2013-09-09

--- ppp-2.4.5+git20130610.orig/scripts/secure-card
+++ ppp-2.4.5+git20130610/scripts/secure-card
@@ -1,4 +1,4 @@
-#!/usr/local/bin/expect -f
+#!/usr/bin/expect -f
 #
 # This  script was  written  by  Jim Isaacson  <jcisaac@crl.com>.  It is
 # designed to work  as a script to use the  SecureCARD(tm) device.  This
